src/AtlassianAssistance.JiraService/Attributes/InsightObjectTypeAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AtlassianAssistance.JiraService.Attributes
+{
+    public sealed class InsightObjectTypeAttribute : Attribute
```

Comment:
Please add notation for use just in class.

Reply:
[5c71fbf](https://github.com/kalhorim/JiraService/commit/5c71fbf7addc091cfeb99bea1cdd44d7b997b18b)